### PR TITLE
Catalog project card

### LINKED ISSUE #4152: [Create] New project card for catalog
### Current Behavior

On https://layer5.io/projects a new project card for `Catalog` needs to be created. This card will link to the new page created under issue #4150 .

### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by goldfishdolphin**
**Description**

This PR fixes #4152

**Notes for Reviewers**
I have not added the link to page yet. @senali-d Please let me know when page is ready so I can add the path to the Link. 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by senali-d — COMMENTED** [review]
@goldfishdolphin  "Service Mesh Patterns" card content overflow from the card.
calalog

**Comment by l5io** [comment]
🚀 Preview for commit 8047d9c791d4328501eb61bb1e3917f0587d4683 at: https://644d7430ff7a0d4e061a82c6--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 45cf0724bfd51f29983d8a490ccd3994c0150d53 at: https://644d7b4d79d552537c2bd687--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1c91e892c157ddac75e4ec9aff109accbae42db0 at: https://644d8f17587f9e5f3247a7ad--layer5.netlify.app

**Review by senali-d — COMMENTED** [review]
@goldfishdolphin Improvement suggestion
Can you center the "Service Mesh Pattern logo" now it aligned left.

image

**Comment by goldfishdolphin** [comment]
Thank you @senali-d. Can you please suggest how to make it center? Do you think it will look good if I make it on the same line as Meshery docker extension?

**Comment by senali-d** [comment]
@goldfishdolphin I think when it align same line there will be some spacing issue. You can refer "Image Hub" or "Meshery" card. It would be great we can do same for the "Service Mesh Pattern logo".

File: src/sections/Projects/Project-grid/projectGrid.style.js [hunk 1]
```diff
@@ -242,10 +242,10 @@ export const ProjectWrapper = styled.div`
         
     }
     .project__card.eight{
-        grid-column: 6/8;
-        grid-row: 6/8;
+        grid-column:1/3;
```
Comment (thoughtlessmind) [inline src/sections/Projects/Project-grid/projectGrid.style.js:245]:
```suggestion
        grid-column: 1/3;
```

File: src/sections/Projects/Project-grid/projectGrid.style.js [hunk 2]
```diff
@@ -403,10 +450,21 @@ export const ProjectWrapper = styled.div`
...
+                margin-bottom:30px;
+            }
+        }
+        .project__card.ten{
+            img{
+                margin-top: 15px;
+                width: 150%;
+            }
+            h5{ 
+                margin-top:30px;
```
Comment (thoughtlessmind) [inline src/sections/Projects/Project-grid/projectGrid.style.js:467]:
```suggestion
                margin-top: 30px;
```

**Comment by thoughtlessmind** [comment]
hey @goldfishdolphin 
for centering the image and also fixing the overflowing issue, I have a suggestion.
you can remove the side padding and then center the image  

image

you can take the reference from the first `Service Mesh Performance` card.
image

**Comment by goldfishdolphin** [comment]
Thank you @thoughtlessmind. I ll try that.

**Comment by vishalvivekm** [comment]
@goldfishdolphin Let's discuss this on websites call. Add this as an agenda item in [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would :)

**Comment by goldfishdolphin** [comment]
@l5io , @vishalvivekm , @senali-d  and @leecalcote 
I used git clone --depth=1 to work on this issue. If I want to make further changes to this branch, I am being asked to pull 10K+ commits. I have removed the repo locally. Is there a way I can work on the branch locally [goldfishdolphin:catalog-project](https://github.com/goldfishdolphin/layer5/tree/catalog-project)?
Thank you

**Comment by senali-d** [comment]
@goldfishdolphin After clone project from your forked repo. Checkout to origin "catalog-project" branch

https://github.com/goldfishdolphin/layer5/tree/catalog-project

**Comment by leecalcote** [comment]
I find the [`gh` CLI](https://cli.github.com) to be quite helpful in terms of checking out branches from various pull requests. For the catalog PR:

```
gh pr checkout 4161
```

**Comment by goldfishdolphin** [comment]
> I find the [`gh` CLI](https://cli.github.com) to be quite helpful in terms of checking out branches from various pull requests. For the catalog PR:
> 
> ```
> gh pr checkout 4161
> ```

Thank you @leecalcote . I will try that.

**Comment by l5io** [comment]
🚀 Preview for commit 60d4f68af15be5ae8618b684d29db7dc8e6df309 at: https://6450b46b931d0f1265f03812--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 363d2a42e5770ec1200926bc16f34e2efe56cc11 at: https://6450e2e5e31c9a3b739171c8--layer5.netlify.app

**Comment by leecalcote** [comment]
@Chadha93 will you assist in minimizing the size of SMI project and maximizing the size of the MeshMap project?

**Comment by leecalcote** [comment]
@ritiksaxena124 should we be using a lighter version of the Catalog icon in dark mode?

**Comment by Chadha93** [comment]
> @Chadha93 will you assist in minimizing the size of SMI project and maximizing the size of the MeshMap project?

Sure.

**Comment by ritiksaxena124** [comment]
> @ritiksaxena124 should we be using a lighter version of the Catalog icon in dark mode?

YES, We should use this [icon](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=12370%3A35382&t=NuOR4A3F7VFY7y6R-1) so that all icons look from the same family

**Comment by Chadha93** [comment]
> @Chadha93 will you assist in minimizing the size of SMI project and maximizing the size of the MeshMap project?

@leecalcote  updated.

**Comment by leecalcote** [comment]
Well, @goldfishdolphin. What was a simple, first contribution has turned into a multi-contributor collaboration. I like your style. 😀

**Comment by l5io** [comment]
🚀 Preview for commit 99ec53557de0a5bbbbb6c57df6c8329156c651e9 at: https://64515fb086a839009a5c9a21--layer5.netlify.app

**Comment by goldfishdolphin** [comment]
> Well, @goldfishdolphin. What was a simple, first contribution has turned into a multi-contributor collaboration. I like your style. 😀

I am learning from experienced contributors. Thank you all for helping me out.

**Comment by l5io** [comment]
🚀 Preview for commit f4ab69e79be89980c9691b399e30057c80c44ded at: https://64558575024ebb226d1dee52--layer5.netlify.app

**Comment by vishalvivekm** [comment]
> > Well, @goldfishdolphin. What was a simple, first contribution has turned into a multi-contributor collaboration. I like your style. grinning
> 
> I am learning from experienced contributors. Thank you all for helping me out.

Great work @goldfishdolphin :tada: :rocket:

**Comment by l5io** [comment]
🚀 Preview for commit 6f0a91025a434fb69fa709ed5a787cd1ffe861e7 at: https://645e0fb5e0d7ab760978bc13--layer5.netlify.app

**Comment by vishalvivekm** [comment]
> @goldfishdolphin @senali-d, here is a description that you can work with in your respective efforts:
> 
> > Introducing Meshery Catalog - the ultimate solution for DevOps engineers looking to optimize their cloud native infrastructure configurations and operational patterns. As a central hub for sharing cloud native infrastructure designs, Meshery Catalog enables you to exchange best practices, reusable templates, and Kubernetes-based operational patterns for multi-cluster Kubernetes clusters and distributed applications.
> > Meshery Catalog functions much like a cloud marketplace, providing a user-friendly interface for browsing, discovering, and sharing configurations and patterns for cloud native infrastructure. With Meshery Catalog, you can easily find and deploy Kubernetes-based infrastructure and tools, making it easy to streamline your cloud native development and operations processes.
> > Meshery Catalog also supports a collaborative environment, where DevOps engineers can share their experiences, feedback, and best practices with others in the community. Import cloud native patterns published by others into your Meshery Server. Benefit from and build upon each pattern by incorporating your own tips and tricks, then publish and share with the community at-large. This facilitates knowledge-sharing and helps to build a strong ecosystem of cloud native infrastructure experts.
> > With Meshery Catalog, you can easily deploy and operate Kubernetes-based infrastructure and distributed applications in a fast, reliable, and secure manner. Whether you're an experienced DevOps engineer or a newcomer to cloud native infrastructure, Meshery Catalog has everything you need to optimize your cloud infrastructure and stay ahead of the curve. Try Meshery Catalog today and discover the power of cloud native infrastructure design patterns and operational templates!

Whoa Whoa Whoa.
@leecalcote 👏 What a Description Good Sir. 🔥 😎 
Thank you so much @leecalcote 

I wonder what @senali-d @goldfishdolphin have got to say on this 👀

**Comment by goldfishdolphin** [comment]
Thank you @leecalcote for the description of the Catalog project. 
I am going to use the first paragraph in the readme file of Layer5.

**Comment by leecalcote** [comment]
Excellent teamwork. :)

**Comment by l5io** [comment]
🚀 Preview for commit 6ffde75aa33721db9573990b81c9c65b0a93d92c at: https://6463870ca8fd0115d37882e2--layer5.netlify.app

**Comment by ayushthe1** [comment]
@goldfishdolphin   Let's discuss this on websites call. Add this as an agenda item in meeting minutes if you would :).
I'll add this item for you if it's ok.

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Thank you, @goldfishdolphin

**Pull request merged**